modules/serverless-event-delivery-failure-codes.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// serverless/develop/serverless-event-delivery.adoc
+
+:_content-type: REFERENCE
+[id="serverless-event-delivery-failure-codes_{context}"]
+= Delivery failure status codes
+
+A channel or broker receiver can respond with the following status codes if an event fails to be delivered:
```

Comment:
Do we really need to explain HTTP status codes?

I'd expect those are well understood standards.

Also, the list is not really complete: https://github.com/knative/specs/blob/main/specs/eventing/data-plane.md#event-acknowledgement-and-delivery-retry

Reply:
IIRC this was documented as a result of discussions in Slack related to https://issues.redhat.com/browse/SRVKE-711
If you think it's unnecessary I'm OK with removing it, I don't think these docs have been touched in about 1.5 years.

Reply:
Perhaps instead of this document, we can say, that `Delivery failure status codes` are standard HTTP (error) codes?

Reply:
Updated: removed the codes bit and added a line about HTTP codes in the intro instead http://file.rdu.redhat.com/abrennan/250822/SRVKE-1277/serverless/develop/serverless-event-delivery.html